INSTALL.md

```diff
@@ -48,6 +48,7 @@ The Helm Chart can be configured using the following parameters (incomplete list
 | `registry.ingress.className`    | The `Ingress` class name                                                                                                                                                                                                                 | `nginx`                                     |
 | `registry.ingress.annotations`    | Annotations to further configure the `Ingress` resource, e.g. for using with `cert-manager`.                                                                                                                                             |                                             |
 | `registry.tenantId`     | TenantId which is the owner of the DTR.                                                                                                                                                                                                  |                                             |
+| `registry.identityProvider`     | Identity provider for the DTR. Possible values are `keycloak` or `cognito`.                                                                                                                                                                                                 |                                             |
```

Comment:
Hello @tunacicek ,
I did not add the idpInternalClientId, because the publicClientId is also not part of this file. Or is the public client id existing in the README.md file with a different name?

Reply:
Hi @peissing ,
i think, public-client-id was forgotten. We need to add it too.